ui/tests/integration/components/auth/form-template-test.js

```diff
@@ -410,5 +414,37 @@ module('Integration | Component | auth | form template', function (hooks) {
       await fillIn(GENERAL.selectByAttr('auth type'), 'oidc');
       assert.dom(GENERAL.inputByAttr('jwt')).doesNotExist();
     });
+
+    test('it should retain role input value when mount path changes', async function (assert) {
```

Comment:
When I was auditing the test coverage, noticed I had missed this one: https://github.com/hashicorp/vault/pull/30527/files#diff-3c3de34a9559a917daca3e4cc85c2d889389631f0580026ea8126d1ff754198eL136-L174 so essentially copied it over here